io-engine/src/lvs/lvol_snapshot.rs

```diff
@@ -44,6 +38,163 @@ use crate::{
 
 use super::{BsError, Lvol, LvsError, LvsLvol};
 
+/// Result for low-level Lvol calls.
+pub type LvolResult = Result<*mut spdk_lvol, Errno>;
+
+///  Traits gives the common snapshot/clone interface for Local/Remote Lvol.
+#[async_trait::async_trait(?Send)]
+pub trait LvolSnapshotOps {
```

Comment:
There will be a corresponding `LogicalVolumeSnapshotOps` for say lvm backend, when it supports snapshot?